dspace/config/dspace.cfg

```diff
@@ -1624,6 +1624,33 @@ google.recaptcha.site-verify = https://www.google.com/recaptcha/api/siteverify
 # checkbox - The "I'm not a robot" Checkbox requires the user to click a checkbox indicating the user is not a robot.
 #google.recaptcha.mode =
 
+#------------------------------------------------------------------#
+#---------------REGISTRATION DATA CONFIGURATION--------------------#
+#------------------------------------------------------------------#
+
+# Configuration for the duration of the token depending on the type
+# the format used should be compatible with the standard DURATION format,
+# but without the prefix `PT`:
+#
+# - PT1H -> 1H // hours
+# - PT1M -> 1M // minutes
+# - PT1S -> 1S // seconds
+#
+eperson.registration-data.token.orcid.expiration = 1H
+eperson.registration-data.token.validation_orcid.expiration = 1H
+eperson.registration-data.token.forgot.expiration = 24H
+eperson.registration-data.token.register.expiration = 24H
+eperson.registration-data.token.invitation.expiration = 24H
+eperson.registration-data.token.change_password.expiration = 1H
+
+# Configuration that enables the schedulable tasks related to the registration
+# The property `enabled` should be setted to true to enable it.
+eperson.registration-data.scheduler.enabled = true
```

Comment:
I'm trying to understand what happens when this is disabled.  It appears, to me, like this is written to manage all registrations into DSpace (not just ORCID).  If this is disabled, will sites no longer be able to register new users?  The description is not clear to me, so I'm trying to understand if this should be enabled by default or not.

Also, I'm wondering if this should be listed **first** among all the new configurations. Will the token settings above still work if this is set to "false"?  If not, then it probably should be listed *before* the tokens.

Reply:
@vins01-4science : Can you please add more details to the comments above this setting to describe the behavior.  Currently, the description is vague as it just says "Configuration that enables the schedulable tasks related to the registration".  It sounds like, from what you are saying, this is enabling automatic cleanup of invalid/expired tokens?  If that's correct, then we need to add that to the description (as I find the phrase "schedulable tasks" to be very vague -- it doesn't say what these tasks are actually doing, and why you might want this setting enabled or disabled.)